src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+function cloneColumn(section) {
+  for (let i = 0; i < section.rows.length; i++) {
+    const row = section.rows[i];
+    const secondCell = row.cells[1];
+    const lastCell = row.cells[row.cells.length - 1];
+    const clonedCell = secondCell.cloneNode(true);
+
+    row.insertBefore(clonedCell, lastCell);
+  }
+}
```

Comment:
This function `cloneColumn` is currently depending on the global scope for the `section` variable. It would be better to pass `section` as a parameter to the function. This would make your function more reusable and easier to test.